sf/draft procurement/stmt of qual.md

```diff
@@ -55,18 +59,27 @@ Play 8 – Choose a Modern Technology Stack
   * Our application is built using open source software solutions at every layer of the stack.
   
 Play 9 – Deploy in a Flexible Hosting Environment  
-  * For at least ¼ of the projects/products for our customers/clients over the last quarter (of a year), any time we provisioned additional servers or instances for production usage, it took less than two hours.
-  * In the last quarter, we have verified that for at least ¼ of the projects/products for our customers/clients, the servers or instances automatically scale based on real-time demand, without any human intervention.
-  * Half or more of the projects/products for our customers/clients the application is currently deployed in a test environment that mirrors the production environment.
+  * We provisioned additional servers or instances for production usage utilizing standard approaches which minimized deployment times over the past year.
+  * Where necessary, and possible, we enabled all servers/instances to automatically scale based on real-time demands, without any human intervention.
+  * At least half of our customer/client projects/products are currently deployed in a test environment that mirrors the production environment.
   
 Play 10 – Automate Testing and Deployments
-  * Currently, half or more of the projects/products for our customers/clients follow software build processes which automatically run automated tests to verify functionality and prevent bugs or broken functionality from being released.
+  * At least half of our customer/client projects/products follow software build processes which automatically run automated tests to verify functionality and prevent bugs or broken functionality from being released.
   
+Play 11 – Manage security and privacy through reusable processes
```

Comment:
- Added in this play which was missing to ensure government compliance with international regulations and protections for consumers